internals/ci/src/cmds/deploy.ts

```diff
@@ -27,6 +27,9 @@ const APP_CONFIGS = {
   "oko-user-dashboard": {
     path: path.join(paths.root, "apps/user_dashboard"),
   },
+  "oko-sandbox-evm": {
+    path: path.join(paths.root, "sandbox/sandbox_evm"),
+  },
```

Comment:
I’m not sure we need to deploy `sandbox_evm` to Vercel. What are your thoughts?

Reply:
First, the reason this was added to the current code was that I needed to deploy `sandbox_evm` for QA purposes.

For EVM features like EIP-712 and SIWE signature UI, using `sandbox_evm` is definitely more convenient, so I think it could be useful in the future as well.

Reply:
got it